guides/cookbook.md

```diff
@@ -1102,6 +1425,53 @@ while (type = reader.next()) {
 ```
 </div>
 
+<div class="tabpane Python" markdown="1">
+#### Using simpleion
+`load/loads` always fully materializes the entire stream. Hence, a sparse read that leverages the
+efficiencies of binary Ion is not possible via this API. With that in mind, it is possible to
+sparsely access the fully-materialized values.
+
+```python
+data = ... # The binary Ion equivalent of the above data.
+values = simpleion.loads(data, single_value=False)
+sum = 0
+for value in values:
+    if u'foo' == value.ion_annotations[0].text:
+        sum += value[u'quantity']
+```
+
+#### Using events
+Unlike `load/loads`, the event-based API may be used for efficient sparse reads over binary Ion
```

Comment:
nit: I would use the same wording on the this first sentence as the one on line 1430, specfically 

"Unlike `load/loads`, the even-based API doe not fully materialize the entire stream and therefore allows for efficient sparse reads over binary Ion" 

or something close to that. Similar sentence structure makes it stick out as the 2 options and emphasizes the differences through repetition.

Reply:
Done.